core/src/App.html

```diff
@@ -237,7 +237,17 @@
       console.info('iframe does not exist, not able to send context.');
       return;
     }
-    const internal = prepareInternalData(config);
+
+    let internal = prepareInternalData(config);
+    const userSettingsGroupName =
+      config.iframe.luigi.currentNode.userSettingsGroup;
+
+    if (userSettingsGroupName) {
+      const userSettingGroups = await LuigiConfig.readUserSettings();
```

Comment:
In case `readUserSettings` returns null, we would have an error when opening the mf, like: 
**TypeError: Cannot read property 'userAccount' of null**
This would happen in case settings are not yet set either through localStorage or through custom readUserSettings not being set.
Probably we can put a warning here that no settings  were found or sth similar to avoid the error:
![Screen Shot 2020-12-16 at 11 51 33](https://user-images.githubusercontent.com/18335489/102341547-47f8d200-3f98-11eb-9196-405b69942cf8.png)

Reply:
Thanks @ndricimrr . I fixed this error at https://github.com/SAP/luigi/pull/1754/files#diff-7a7028de38452fdd8eef4c66743213004e94b3127d3e088c63fb65b09d587f7aR256-R258. Please have a look again.